src/Consts/pathData.js

```diff
@@ -66,6 +57,11 @@ const pathData = [
     text: 'Konkurs',
     label: 'XV zjazd dobrych chłopaków',
   },
+  {
+    path: '/contests/:contestId/classes/:classId/leaderboard',
+    text: 'Leaderboard',
+    label: 'wybrany contest',
+  },
```

Comment:
I think it cannot work ok because of path with variable places `:contestId` `:classId`